bcipy/helpers/triggers.py

```diff
@@ -638,3 +640,195 @@ def read_triggers(triggers_file: TextIO) -> List[Tuple[str, str, float]]:
+        ----------
+        None
+
+        Returns
+        -------
+            None
+        """
+
+        for trigger in self.triggers:
+            self.file.write(f'{trigger.label} {trigger.type.value} {trigger.time}\n')
+
+        self.triggers = []
+
+    @staticmethod
+    def read_text_file(path: str) -> Tuple[List[List[str]], float]:
+        if not path.endswith('.txt') or not os.path.exists(path):
+            raise FileNotFoundError(f"Valid triggers .txt file not found at [{path}]."
+                                    "\nPlease rerun program.")
+
+        triggers_list = []
+        with open(path) as raw_txt:
+            for line in raw_txt:
+                line_split = line.split()
+                triggers_list.append(line_split)
+
+        # find offset values in list and Return
+        try:
+            if triggers_list[0][1] == TriggerType.OFFSET.value:
+                offset = float(triggers_list[0][2])
+                triggers_list.pop(0)
+            else:
+                offset = 0.0
+        except Exception as e:
+            raise BciPyCoreException(
+                f'Invalid triggers.txt format error=[{e}] triggers=[{triggers_list}]')
+        return triggers_list, offset
+
+    @staticmethod
+    def load(path: str,
+             offset: Optional[float] = 0.0,
+             exclusion: Optional[List[TriggerType]] = None) -> List[Trigger]:
+        """
+        Loads a list of triggers from a .txt of triggers.
+
+        Exclusion based on type only (ex. exclusion=[TriggerType.Fixation])
+
+        1. Checks if .txt file exists at path
+        2. Loads the triggers data as a list of lists
+        3. If offset provided, adds it to the time as float
+        4. If exclusion provided, filters those triggers
+        5. Casts all loaded and modified triggers to Trigger
+        6. Returns as a List[Triggers]
+
+        Parameters
+        ----------
+        path (str): name or file path of .txt trigger file to be loaded.
+            Input string must include file extension (.txt).
+        offset (Optional float): if desired, time offset for all loaded triggers,
+            positive number for adding time, negative number for subtracting time.
+        exclusion (Optional List[TriggerType]): if desired, list of TriggerType's
+            to be removed from the loaded trigger list.
+
+        Returns
+        -------
+            List of Triggers from loaded .txt file with desired modifications
+        """
+
+        # Checking for file with given path, with or without .txt
+        triggers_list, system_offset = TriggerHandler.read_text_file(path)
+
+        try:
+            if exclusion:
+                for trigger_type in exclusion:
+                    triggers_list[:] = [item for item in triggers_list if not trigger_type.value == item[1]]
+
+            # apply system and provided offsets
+            for item in triggers_list:
+                item[2] = float(item[2]) + offset + system_offset
```

Comment:
One thing to note is that we may not always want to apply the system_offset. In the recent refactor of the acquisition code the timestamps given in the triggers.txt file are using the same clock as those in the raw data, so if we want to relate the two we don't want to apply the offset. But maybe that distinction should be handled elsewhere.